lib/src/markdown.pipe.ts

```diff
@@ -8,6 +8,7 @@ export type MarkdownPipeOptions = ParseOptions & RenderOptions;
 
 @Pipe({
   name: 'markdown',
+  standalone: true,
```

Comment:
this is part of a complete rewrite to the standalone see more https://github.com/jfcere/ngx-markdown/pull/484